pkg/config/lvmd/lvmd_test.go

```diff
@@ -0,0 +1,142 @@
+package lvmd
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+)
+
+const (
+	microShiftTestCfg = `./test/config.yaml`
+	lvmdTestConfig    = `./test/lvmd.yaml`
+)
+
+const (
+	testConfig     = "./test/config.yaml"
+	testLvmdConfig = "./test/lvmd.yaml"
+)
+
+var spareGb10 = uint(10)
+var stripes = uint(2)
+
+func TestLvmd_withDefaults(t *testing.T) {
+	type fields struct {
+		DeviceClasses []*DeviceClass
+		SocketName    string
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   *Lvmd
+	}{
+		{
+			name: "sets defaults",
+			fields: fields{
+				SocketName: defaultSockName,
+				DeviceClasses: []*DeviceClass{
+					{
+						Name:        "default",
+						Default:     true,
+						VolumeGroup: defaultRHEL4EdgeVolumeGroup,
+						SpareGB:     func() *uint64 { u := uint64(defaultSpareGB); return &u }(),
+					},
+				},
+			},
+			want: &Lvmd{
+				SocketName: defaultSockName,
+				DeviceClasses: []*DeviceClass{
+					{
+						Name:        "default",
+						Default:     true,
+						VolumeGroup: defaultRHEL4EdgeVolumeGroup,
+						SpareGB:     func() *uint64 { u := uint64(defaultSpareGB); return &u }(),
+					},
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			l := &Lvmd{
+				DeviceClasses: tt.fields.DeviceClasses,
```

Comment:
Do we need to populate the `Lvmd` instance here in order to get valid defaults? It looks like `withDefaults` always returns the same data.

Reply:
I've removed this test.  withDefaults returns a static type, which would be checked at compile time.  No point in testing for it as well.